tests/credentials.json

```diff
@@ -14,7 +14,7 @@
         "credentials": {
             "apiKey": "ZZZZZZZZZZ",
             "apiToken": "MMMMMMMMMM",
-            "base_uri": "https://openwhisk.ng.bluemix.net/api/v1",
+            "base_uri": "https://openwhisk.ng.bluemix.net/api/v1"
```

Comment:
please use a more generic uri (i.e. no ibm stuff) 
maybe change to "base_uri": "https://api.service-y.com/v1"